core/src/memory.rs

```diff
@@ -36,10 +42,46 @@ const GEAR_PAGE_SIZE: usize = 0x1000;
 /// Number of gear pages in one wasm page
 const GEAR_PAGES_IN_ONE_WASM: u32 = (WASM_PAGE_SIZE / GEAR_PAGE_SIZE) as u32;
 
-pub use gear_core_errors::MemoryError as Error;
+/// Buffer for gear page data.
+/// TODO: impl self debug
```

Comment:
TODO line must be comment, not doc string

Reply:
Ok done